test/services/datasources/datasourceRuns.test.js

```diff
@@ -37,7 +37,8 @@ class MockSyncerWithData extends Syncer {
 	}
 }
 
-describe('datasourceRuns service', () => {
+describe('datasourceRuns service', function test() {
```

Comment:
Is there a reason to prefer a named function over a anonymous function (as used before) in this case?

Reply:
I need the this context for the timeout.